extensions/identity-hub-verifier/src/main/java/org/eclipse/dataspaceconnector/identityhub/verifier/IdentityHubCredentialsVerifier.java

```diff
@@ -92,7 +93,12 @@ public Result<Map<String, Object>> getVerifiedCredentials(DidDocument didDocumen
 
         var claims = extractClaimsFromCredential(verifiedCredentials);
 
-        return Result.success(claims);
+        if (claims.failed()) {
+            monitor.severe(String.format("Credentials verification failed: %s", claims.getFailureDetail()));
```

Comment:
```suggestion
            monitor.severe(() -> String.format("Credentials verification failed: %s", claims.getFailureDetail()));
```
best practice to avoid computing string if logging disabled at that level